app/views/groups/_form.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= form_with(model: group) do |form| %>
+  <% if group.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(group.errors.count, "error") %> prohibited this group from being saved:</h2>
+
+      <ul>
+        <% group.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
- [ ]  We don't need to add any logic inside the .erb file, we use the helper folders to create methods with logic and called inside the .erb file. You can refer to this comment to modify the files that have logic inside the .erb file.

Read [this](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make) for more information